scopes/react/react/typescript/tsconfig.build.json

```diff
@@ -15,7 +15,13 @@
     "moduleResolution": "node",
     "esModuleInterop": true,
     "rootDir": ".",
-    "resolveJsonModule": true
+    "resolveJsonModule": true,
+    "paths": {
+      "react": [
+        "./node_modules/@types/react",
+        "../node_modules/@types/react"
+      ]
```

Comment:
this seems stable - it tells TS to look for `react` at the current folder `node_modules` or the parent `node_modules`.

this makes sense in the current capsule structure:
```
shared dir
- node_modules <-- shared modules
-- @types 
--- react <-- resolved location (fallback)
- target-dir
-- tsconfig.json (the '.' path)
-- node_modules
--- @types
---- react <-- resolved location (preferred)
```

Reply:
real world example:
<img width="351" alt="Screen Shot 2022-05-08 at 20 36 29" src="https://user-images.githubusercontent.com/5400361/167308579-cc591fc7-2e3d-48eb-8419-7b2d11567cc9.png">